prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Cristian Quinones';
+    console.log('value of fullName;', fullName);
+    var numberofPets = '1';
+    console.log('value of numberofPets;', numberofPets);
+    var likesVegetables = 'true';
+    console.log('value of likesVegetables;', likesVegetables);
+    var nothing = 'null'
+    console.log('value of null;', null);
```

Comment:
For numbers, booleans, and `null`, we want to leave out the quotes so that the JavaScript engine doesn't interpret them as strings.